### DISCUSSION TITLE
스터디 8주차 문제

### DISCUSSION THREAD

**Discussion by DaleSeo** [DaleStudy/leetcode-study#139] [공지사항]
> [!NOTE]
> 풀이 기간: 2024년 6/16(일)~6/22(토)

## Kth Smallest Element In a Bst
- 문제: https://leetcode.com/problems/kth-smallest-element-in-a-bst/
- 풀이: https://www.algodale.com/problems/kth-smallest-element-in-a-bst/
- 위치: https://github.com/DaleStudy/leetcode-study/tree/main/kth-smallest-element-in-a-bst

## Construct Binary Tree From Preorder And Inorder Traversal
- 문제: https://leetcode.com/problems/construct-binary-tree-from-preorder-and-inorder-traversal/
- 풀이: https://www.algodale.com/problems/construct-binary-tree-from-preorder-and-inorder-traversal/
- 위치: https://github.com/DaleStudy/leetcode-study/tree/main/construct-binary-tree-from-preorder-and-inorder-traversal

## Combination Sum
- 문제: https://leetcode.com/problems/combination-sum/
- 풀이: https://www.algodale.com/problems/combination-sum/
- 위치: https://github.com/DaleStudy/leetcode-study/tree/main/combination-sum

## Word Search
- 문제: https://leetcode.com/problems/word-search/
- 풀이: https://www.algodale.com/problems/word-search/
- 위치: https://github.com/DaleStudy/leetcode-study/tree/main/word-search

## Implement Trie Prefix Tree
- 문제: https://leetcode.com/problems/implement-trie-prefix-tree/
- 풀이: https://www.algodale.com/problems/implement-trie-prefix-tree/
- 위치: https://github.com/DaleStudy/leetcode-study/tree/main/implement-trie-prefix-tree

**Comment by WhiteHyun** [comment]
`Combination Sum` 문제 URL이 풀이와 동일합니다!

**Reply by Invidam** [reply]
> `Combination Sum` 문제 URL이 풀이와 동일합니다!

수정해버렸어요!
